From 6b62bca4c9555efa1c8afebecc4b48b88d2603d9 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 8 Sep 2018 22:40:46 +0300
Subject: [PATCH 234/249] lineage: Remove unused qcom repositories

* brctl/ebtables were used by radish which is deprecated
* system/qcom is not used since we do not use anything
  that needs libqsap_sdk since oreo

Change-Id: I0a663ab5916b796d6dff041b70bb75a71c4eb233
---
 snippets/lineage.xml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index a98719a..47c0226 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -57,9 +57,7 @@
   <project path="external/ant-wireless/ant_native" name="LineageOS/android_external_ant-wireless_ant_native" />
   <project path="external/ant-wireless/ant_service" name="LineageOS/android_external_ant-wireless_ant_service" />
   <project path="external/ant-wireless/antradio-library" name="LineageOS/android_external_ant-wireless_antradio-library" />
-  <!--<project path="external/brctl" name="LineageOS/android_external_brctl" />-->
   <project path="external/connectivity" name="LineageOS/android_external_connectivity" />
-  <!--<project path="external/ebtables" name="LineageOS/android_external_ebtables" />-->
   <project path="external/json-c" name="LineageOS/android_external_json-c" />
   <!--<project path="external/powertop" name="LineageOS/android_external_powertop" />-->
   <!--<project path="hardware/qcom/audio-caf/apq8084" name="LineageOS/android_hardware_qcom_audio" groups="qcom,qcom_audio,pdk" revision="lineage-16.0-caf-8084" />-->
@@ -96,7 +94,6 @@
   <!--<project path="hardware/qcom/vr" name="LineageOS/android_hardware_qcom_vr" />-->
   <project path="hardware/qcom/wlan-caf" name="LineageOS/android_hardware_qcom_wlan" groups="qcom_wlan,pdk" revision="lineage-16.0-caf" />
   <project path="hardware/ril-caf" name="LineageOS/android_hardware_ril" groups="pdk" revision="lineage-16.0-caf" />
-  <!--<project path="system/qcom" name="LineageOS/android_system_qcom" groups="qcom" />-->
   <project path="system/tools/dtbtool" name="LineageOS/android_system_tools_dtbtool" />
   <project path="vendor/codeaurora/telephony" name="LineageOS/android_vendor_codeaurora_telephony" />
   <!--<project path="vendor/nxp/interfaces/opensource/nfc" name="LineageOS/android_vendor_nxp_interfaces_opensource_nfc" />-->
-- 
2.17.1

